### PR TITLE
4506 Reports page is blank if the reports aren't configured

### LINKED ISSUE #4506: Reports page is blank if the reports aren't configured
## What went wrong? 😲

The empty sections render while the page is loading, and then once it realises I don't have reports configured, it goes completely blank?

https://github.com/user-attachments/assets/a9c0a054-8b40-48fd-a17b-6cd21b2a7f93

## Expected behaviour 🤔

Would be nice to see the sections still, or the nothing here component, or something...

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Ensure you don't have any reports with the `Report` context in your DB
2. Go to reports page
3. Sections load for half a second, then blank

## Your environment 🌱

- Open mSupply Version: 2.2
- Legacy mSupply Central Server Version:

- Platform:

- Database type:

### PR DESCRIPTION
**Pull request by roxy-dao**
Fixes #4506

# 👩🏻‍💻 What does this PR do?
Add Nothing here with a message to contact support about setting up reports if there are no reports.

# 🧪 Testing

- [ ] Don't have any reports set up (not even program reports)
- [ ] Go to report page
- [ ] See message

# 📃 Documentation

- [x] **Part of an epic**: documentation will be completed for the feature as a whole
- [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1.
  2.

### CODE REVIEW

File: client/packages/common/src/intl/locales/en/reports.json [hunk 1]
```diff
@@ -8,5 +8,6 @@
     "label.monthly-consumption-look-back-period": "Monthly consumption look back period",
     "label.stocktake-frequency": "Stocktake frequency",
     "label.threshold-for-overstock": "Threshold for overstock",
-    "label.threshold-for-understock": "Threshold for understock"
+    "label.threshold-for-understock": "Threshold for understock",
+    "message.contact-support": "Please contact support at support@msupply.foundation about configuring reports."
```
Comment (CarlosNZ) [inline client/packages/common/src/intl/locales/en/reports.json:12]: The "support at support" is a bit weird. Just looking through the other examples, seems we just have the email.

An existing example: "The mSupply Central Server needs to be upgraded to a newer version before you can connect - please contact support@msupply.foundation"

Also -- don't need the trailing "." -- seems we don't put a period for single sentence messages.

File: client/packages/reports/src/ListView/ListView.tsx [hunk 2]
```diff
@@ -63,6 +64,15 @@ export const ListView = () => {
     },
     [navigate]
   );
+
+  if (
+    !store?.preferences?.omProgramModule ||
+    !stockAndItemReports?.length ||
+    !expiringReports?.length
```
Comment (CarlosNZ) [inline client/packages/reports/src/ListView/ListView.tsx:71]: This fails if *any* of the conditions are `true`, so I'm getting the "Nothing here" even when I have reports enabled.

I think they should probably be `&&`, right?

And maybe doesn't need `!store?.preferences?.omProgramModule`? Would you ever hide the reports even though they are available?

**Review by CarlosNZ — CHANGES_REQUESTED** [review]
Cool, see comments.

File: client/packages/reports/src/ListView/ListView.tsx (hunk 2 above)
Comment (roxy-dao) [reply client/packages/reports/src/ListView/ListView.tsx:71]: I guess not since the script for program pre-configs the reports too...

**Review by CarlosNZ — APPROVED** [review]

**Comment by CarlosNZ** [comment]
Nice, thanks for the fixes @roxy-dao

**Pull request merged**